client/packages/invoices/src/Returns/OutboundDetailView/DetailView.tsx

```diff
@@ -81,13 +87,14 @@ export const OutboundReturnsDetailView: FC = () => {
               supplierId={data.otherPartyId}
               returnId={data.id}
               initialItemId={itemId}
+              modalMode={mode}
             />
           )}
 
           <Toolbar />
           <DetailTabs tabs={tabs} />
           <Footer />
-          <SidePanel />
+          {/* <SidePanel /> */}
```

Comment:
Why is this commented out?

Reply:
I dunno, I think something got mixed up when pulling in from base branch at some point. Will fix.